src/gui/application.cpp

```diff
@@ -186,7 +186,8 @@ Application::Application(int &argc, char **argv)
     setAttribute(Qt::AA_UseHighDpiPixmaps, true);
 
     auto confDir = ConfigFile().configPath();
-    if (!QFileInfo(confDir).exists()) {
+    if (confDir.endsWith('/')) confDir.chop(1);  // macOS 10.11.x does not like trailing slash for rename/move.
+    if (!QFileInfo(confDir).isDir()) {
```

Comment:
and if it is a file?
You can't do the move in this case anyways either. So this changes change nothing.

Reply:
I think it was always a dir, even in old mirall versions. It's better to use the more explicit functions so that the code states the intention (further down correctly isDir is used)